packages/tools/installer/scripts/installer

```diff
@@ -634,20 +635,29 @@ do_poweroff() {
 }
 
 # setup needed variables
-INSTALLER_VERSION="0.2.8"
+INSTALLER_VERSION="0.3.0"
```

Comment:
@trsqr feel free to make another PR with improvements that you see fit for the installer. I'd rather stuff just get done/fixed rather than having long discussions about it :+1: Keep up the great work